roles/lnls-ans-role-qt/defaults/main.yml

```diff
@@ -21,7 +21,8 @@ qt_src_checksum: "{{ pkg_version_qt_src_checksum }}"
 qt_src_remote_script: "qt-opensource-linux-x64-{{ qt_src_version }}.run"
 
 # Qt URL downlaod script
-qt_src_download_url: "https://download.qt.io/archive/qt/{{ qt_src_version_reduced }}/{{ qt_src_version }}/{{ qt_src_remote_script }}"
+# qt_src_download_url: "https://download.qt.io/archive/qt/{{ qt_src_version_reduced }}/{{ qt_src_version }}/{{ qt_src_remote_script }}"
```

Comment:
I think we should use the previous sensible defaults for this. If wanted we can set this variable elsewhere and ansible will override it automatically